src/main/java/org/cancogenvirusseq/muse/exceptions/submission/MissingDataException.java

```diff
@@ -19,7 +19,7 @@ public String getMessage() {
 
   public Map<String, Object> getErrorInfo() {
     return Map.of(
-        "isolateInFileMissingInTsv", isolateInFileMissingInTsv,
-        "isolateInRecordMissingInFile", isolateInRecordMissingInFile);
+        "fastaHeaderInFileMissingInTsv", fastaHeaderInFileMissingInTsv,
+        "fastaHeaderInRecordMissingInFile", fastaHeaderInRecordMissingInFile);
```

Comment:
this will need UI update which is my next ticket

Reply:
:+1: